### ISSUE TITLE
Gets locked in 'shift' mode

### ISSUE THREAD

**Issue by its-all-waves** [aioutecism/amVim-for-VSCode#322]
Hello! First off, thank you for this plug. Been loving it!

I can't seem to reproduce the issue intentionally, but probably twice per day, I have to do a full quit and relaunch of vscode, or uninstall and re-install this plug, to get normal behavior back.

**The Problem**
I'm typing along, then I escape, and now every J press is truly a shift+J press, concatenating the next line to the current line. Pretty darn sure other shift commands are being called when other buttons are pressed, as well. I just restarted, so I can't test the latter at the moment. 

In short, it's as if I'm entering a sort of 'shift-lock' mode, where the shift key is being held for every vim motion I execute. (The problem is definitely not my keyboard, as it behaves as expected using any other application at the time this problem occurs with amVim.)

Version: 1.78.2 (Universal)
Commit: b3e4e68a0bc097f0ae7907b217c1119af9e03435
Date: 2023-05-10T14:44:45.204Z
Electron: 22.5.2
Chromium: 108.0.5359.215
Node.js: 16.17.1
V8: 10.8.168.25-electron.0
OS: Darwin arm64 21.6.0
Sandboxed: Yes

Thank you!

**Comment by alisonatwork** [comment]
Glad you seemed to have figured it out! And I'm happy you're enjoying the extension too. I haven't had as much time to work on it as I would hope recently, but feel free to open more bug reports or request for improvement, hopefully I or others in the community can keep on improving 👍
